scripts/generate_dump

```diff
@@ -235,14 +250,15 @@ save_cmd_all_ns() {
 #  None
 ###############################################################################
 copy_from_docker() {
+    trap 'handle_error $? $LINENO' ERR
     local start_t=$(date +%s%3N)
     local end_t=0
     local docker=$1
     local filename=$2
     local dstpath=$3
     local timeout_cmd="timeout --foreground ${TIMEOUT_MIN}m"
 
-    local touch_cmd="sudo docker exec -i ${docker} touch ${filename}"
```

Comment:
>-i [](http://example.com/codeflow?start=38&length=2)

Could you detect the environment and add `-i` as the default.

Reply:
But why do we require -i for a touch command. Infact i've looked at other docker exec commands used in this script. All of them just write to stdout, so i don't see why -i has to be retained

Reply:
Let me know what do you think about this